### PR TITLE
Fix two issues with interactive window

### PR DESCRIPTION
**Pull request by rchiodo**
Fixes: #7648, #7658

### CODE REVIEW

File: src/client/datascience/interactive-window/interactiveWindow.ts [hunk 1]
```diff
@@ -510,15 +511,31 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
...
             notebookEditor.revealRange(notebookRange, NotebookEditorRevealType.Default);
+
+            // Also add a decoration to make it look highlighted (peek background color)
+            if (decorationType) {
+                notebookEditor.setDecorations(decorationType, notebookRange);
+
+                // Fire another timeout to dispose of the decoration
+                setTimeout(() => {
+                    decorationType.dispose();
+                }, 2000);
```
Comment (DonJayamanne) [inline src/client/datascience/interactive-window/interactiveWindow.ts:537]: 1. Hmm, feels hackis, what if i select another cell or run multiple cells.
Now we'd have multiples with the same style.

2. Why are we disposing this after 2s, why don't we just add this to the list of disposables or another list and clear that (dispose it) when we reveal another cell.

**Comment by DonJayamanne** [comment]
Not sold on the cell highlighting, 
will the highlithing go away when users click on the cells or select another cell?

File: src/client/datascience/interactive-window/interactiveWindow.ts [hunk 2]
```diff
@@ -510,15 +511,31 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
...
+            }
         }, 200); // Rendering output is async so the output is not guaranteed to immediately exist
```
Comment (DonJayamanne) [inline src/client/datascience/interactive-window/interactiveWindow.ts:539]: I think we should fix this, filed a bug https://github.com/microsoft/vscode-jupyter/issues/7686

File: src/client/datascience/interactive-window/interactiveWindow.ts (hunk 1 above)
Comment (rchiodo) [reply src/client/datascience/interactive-window/interactiveWindow.ts:537]: That's the behavior from before. Just replicating it. It flashed for 2 seconds.

**Comment by rchiodo** [comment]
> Not sold on the cell highlighting,
> will the highlithing go away when users click on the cells or select another cell?

Highlighting will go away after 2 seconds.

**Comment by DonJayamanne** [comment]
> Highlighting will go away after 2 seconds.

Why not 5s or 10s or until user runs another cell or directly interacts with interactive window?

**Comment by DonJayamanne** [comment]
> That's the way it used to work. Not changing the old behavior.

Ok, thats bad. From what you're saying we've some how removed some code that did this same thing early this month.
Sorry, i coudln't find any code or css that was releated in ths PR, hence assumed this was new & some other part of the PR was existing behaviour.

**Comment by rchiodo** [comment]
> > That's the way it used to work. Not changing the old behavior.
> 
> Ok, thats bad. From what you're saying we've some how removed some code that did this same thing early this month.
> Sorry, i coudln't find any code or css that was releated in ths PR, hence assumed this was new & some other part of the PR was existing behaviour.

That code was in the webview CSS/implementation.

**Comment by rchiodo** [comment]
Here's a link to the original code. Sorry I was wrong, it was only a second.

https://github.com/microsoft/vscode-jupyter/blob/f5c89293163be4c90341104ae4aa151610027d98/src/datascience-ui/history-react/interactiveCell.tsx#L86

**Pull request merged**